cypress/support/components/EditDialogModal.js

```diff
@@ -131,26 +131,26 @@ export function VerifyThereAreNoChatEditControls(userMessage, botMessage) {
 export function LabelTextAsEntity(text, entity, itMustNotBeLabeledYet = true) {
   function LabelIt() {
     // This actually works if text is a word or a phrase.
-    cy.Get('body').trigger('Test_SelectWord', { detail: text });
-    cy.Get('[data-testid="entity-picker-entity-search"]').type(`${entity}{enter}`);
+    cy.Get('body').trigger('Test_SelectWord', { detail: text })
+    cy.Get('[data-testid="entity-picker-entity-search"]').type(`${entity}{enter}`)
   }
 
   if (itMustNotBeLabeledYet) LabelIt()
   else {
     // First make sure it is not already labeled before trying to label it.
     cy.WaitForStableDOM()
     cy.Enqueue(() => {
-      var found = false;
-      var elements = Cypress.$('[data-testid="token-node-entity-value"] > span > span');
+      var found = false
+      var elements = Cypress.$('[data-testid="token-node-entity-value"] > span > span')
 
       // If you need to find a phrase, this part of the code will fail, 
       // you will need to upgrade this code in that case.
       var element = elements.find(element => element.innerText === text)
       if (element) {
-        found = Cypress.$(element).parents('.cl-entity-node--custom').find(`[data-testid="custom-entity-name-button"]:contains('${entity}')`).length == 0;
+        found = Cypress.$(element).parents('.cl-entity-node--custom').find(`[data-testid="custom-entity-name-button"]:contains('${entity}')`).length == 0
```

Comment:
Don't need to change but try to use strict equals (`===`) whenever possible. It helps avoid the implicit type coercion that javascript does and is often what you intend to check. There are some cases you want type coercsion like checking for `null` && `undefined` using a single `==` but it's rare.

Don't think applies in this scenario since `find` probably guarantees to return an array but the canonical examples are:
```
undefined == null
// true
false == 0
// true
```
These cases could lead to unintended behavior of the app.

Reply:
Okay, got it.